### PR TITLE
Tests for products

### PR DESCRIPTION
**Pull request by VolodymyrTrach**

### CODE REVIEW

File: __tests__/news/news.mutation.new.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  ],
+  text: [
+    { lang: 'qe', value: 'ada' },
+    { lang: 'da', value: 'ada' },
+  ],
+};
+
+describe('test news mutations', () => {
+  test('#1 should not throw', () => {
+    expect(newsMutation.addNews).not.toThrow();
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply __tests__/news/news.mutation.new.test.js]: Already done by Vova Vasylkiv

File: __tests__/news/news.query.test.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply __tests__/news/news.query.test.js]: Already done by Vova Vasylkiv

File: __tests__/news/news.mutation.new.test.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-undef */
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply __tests__/news/news.mutation.new.test.js]: Already done by Vova Vasylkiv

File: __tests__/product/product.mutation.test.js [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+let sameNameProductId;
+
+describe('Product mutations', () => {
+  test('#1 Should add new product', async () => {
+    const createProduct = await client.mutate({
+      mutation: gql`
+        mutation($product: ProductInput!) {
+          addProduct(product: $product) {
+            ... on Product {
+              _id
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply __tests__/product/product.mutation.test.js:23]: added more expects for: get, update and delete

File: __tests__/product/product.query.test.js [hunk 5]
```diff
@@ -4,8 +4,7 @@ const client = require('../../utils/apollo-test-client');
 const {
   badProductId,
   newProduct,
-  expectedResult,
-} = require('./product.variables');
+} = require('../../tests-variables/product.variables');
```
Comment (Alan18081) [inline __tests__/product/product.query.test.js:7]: There is no need to move `product` variables to another folder, please, just hold product variables inside `product` folder, in tests

File: __tests__/product/product.mutation.test.js [hunk 6]
```diff
@@ -6,7 +6,7 @@ const {
   newProduct,
   productForUpdate,
   sameNameForUpdate,
-} = require('./product.variables');
+} = require('../../tests-variables/product.variables');
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply __tests__/product/product.mutation.test.js]: moved and deleted separate folder

File: __tests__/product/product.query.test.js (hunk 5 above)
Comment (VolodymyrTrach) [reply __tests__/product/product.query.test.js]: moved and deleted separate folder

**Review by Crash1212 — APPROVED** [review]

**Review by Alan18081 — APPROVED** [review]

**Pull request merged**